src/praisonai/tests/unit/test_approval_agent_integration.py

```
@@ -50,18 +50,15 @@
         result = agent.execute_tool("execute_command", {"command": "echo 'Hello from agent-executed command!'"})
         
-        if result.get('success'):
-            print(f"✅ Command executed successfully: {result['stdout']}")
-        elif result.get('approval_denied'):
-            print(f"❌ Command was denied by approval system: {result['error']}")
+        if result and "Hello from agent-executed command!" in str(result):
+            print("✅ Command executed successfully with approval")
         else:
-            print(f"⚠️ Command failed for other reasons: {result}")
+            print("❌ Command execution failed:", result)
+            assert False, f"Command execution failed: {result}"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)